routes/api.js

```diff
@@ -8,7 +8,7 @@ exports.appendToFile = function (filePath, dataStr) {
   var stream = fs.createWriteStream(filePath, {
     flags: "a",
     encoding: "utf8",
-    mode: 0666,
+    mode: "0666",
```

Comment:
what's the reason for this change?

Reply:
Weird... The reason for the change originally was because there was a warning about legacy octals not being supported, but I'm not seeing it when I hit the route.

I referenced this stackoverflow post as a fix - https://stackoverflow.com/questions/23609042/how-to-avoid-octal-literals-are-not-allowed-in-strict-mode-with-createwritestr

Reply:
So running `jest __tests__/integration/app.test.js` reproduces the legacy octal error I mentioned up here and the fix is to wrap the value in a string.

Reply:
nevermind, I just fixed this by implementing the changes from [#35](https://github.com/rithmschool/numbers_api/pull/35) to this pull request. I'll go ahead and close #35 out @mmmaaatttttt!